zcbor/zcbor.py

```diff
@@ -581,28 +581,7 @@ def set_quantifier(self, quantifier):
         raise ValueError("invalid quantifier: %s" % quantifier)
 
     def convert_min_max(self):
-        defines = {
-            (2**64) - 1: "UINT64_MAX",
-            (2**63) - 1: "INT64_MAX",
-            (2**32) - 1: "UINT32_MAX",
-            (2**31) - 1: "INT32_MAX",
-            (2**16) - 1: "UINT16_MAX",
-            (2**15) - 1: "INT16_MAX",
-            (2**8) - 1: "UINT8_MAX",
-            (2**7) - 1: "INT8_MAX",
-            -(2**63): "INT64_MIN",
-            -(2**31): "INT32_MIN",
-            -(2**15): "INT16_MIN",
-            -(2**7): "INT8_MIN",
-        }
-        if self.min_value in defines:
-            self.min_value = defines[self.min_value]
-        if self.max_value in defines:
-            self.max_value = defines[self.max_value]
-        if self.min_size in defines:
-            self.min_size = defines[self.min_size]
-        if self.max_size in defines:
-            self.max_size = defines[self.max_size]
+        pass
```

Comment:
nothing wrong with this line. Is the commit message correct? "Do conversion... only when doing code generation. It breaks data validation if applied there." ?

Reply:
Yes, but maybe it's ambiguous. The conversion was always applied, but that doesn't work for validation/conversion, since the Python code doesn't understand e.g. "UINT32_MAX". The commit moves the function from the base type to the child type that is only used for code generation.